akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/Main.scala

```diff
@@ -1,14 +1,14 @@
 package com.tapiporla.microservices.retrievers
 
 import akka.actor.{ActorSystem, Props}
-import com.tapiporla.microservices.retrievers.indices.ibex35.Ibex35Coordinator
+import com.tapiporla.microservices.retrievers.indices.stock.StockCoordinator
 
 object Main {
 
   def main(args: Array[String]): Unit = {
 
     val system = ActorSystem("retrievers")
-    val testRetriever = system.actorOf(Props[Ibex35Coordinator])
+    val testRetriever = system.actorOf(Props[StockCoordinator])
```

Comment:
I totally discourage using this ```Props``` apply. It implies that the actor you want to create has a class constructor with zero arguments. I prefer to define a ```def apply(): StockCoordinator``` at the companion object for ```StockCoordinator``` and then use ```Props(StockCoordinator())```

Reply:
Sure, I will follow the Akka recoomendation:

object StockCoordinator {
  def props(): Props = Props(new StockCoordinator())
}

And invoke with: StockCoordinator.props()